services/personal/api.go

```diff
@@ -3,7 +3,7 @@ package personal
 import (
```

Comment:
is the personal api still use? should it not belong to account?

Reply:
Yes, the client is still using the personal API and we cannot remove it, or not yet. In one of the follow-up PRs, a sign function will become part of the accounts management package.